ci-operator/step-registry/telco5g/log-gather/telco5g-log-gather-commands.sh

```diff
@@ -82,33 +81,26 @@ for node in $(oc get node -oname | grep cnf); do
   get_sos_report ${node##*/} || true
 done
 
-echo "Running gather-pao for T5CI_VERSION=${T5CI_VERSION}"
-
-if [[ "$T5CI_VERSION" == "4.13" ]]; then
-    export CNF_BRANCH="master"
-elif [[ "$T5CI_VERSION" == "4.14" ]]; then
-    export CNF_BRANCH="master"
-elif [[ "$T5CI_VERSION" == "4.15" ]]; then
-    export CNF_BRANCH="master"
+echo "************ telco5g gather-pao commands ************"
+if [[ "$T5CI_VERSION" == "4.11" || "$T5CI_VERSION" == "4.12" || "$T5CI_VERSION" == "4.13" ]]; then
+    echo "Running gather-pao for T5CI_VERSION=${T5CI_VERSION}"
+
+    if [[ "$T5CI_VERSION" == "4.11" ]]; then
+        pao_mg_tag="4.11"
+    elif [[ "$T5CI_VERSION" == "4.12" ]]; then
+        pao_mg_tag="4.12"
+    elif [[ "$T5CI_VERSION" == "4.13" ]]; then
+        pao_mg_tag="4.13"
```

Comment:
Looks like it's consistent here? Maybe then just
```bash
pao_mg_tag="$T5CI_VERSION"
```

Reply:
yeah, good point, I will update after pj rehearse job passes.